aws_lambda_powertools/utilities/data_classes/vpc_lattice.py

```diff
@@ -4,15 +4,15 @@
     BaseHeadersSerializer,
     HttpApiHeadersSerializer,
 )
-from aws_lambda_powertools.utilities.data_classes.common import BaseProxyEvent
+from aws_lambda_powertools.utilities.data_classes.common import BaseProxyEvent, DictWrapper
 from aws_lambda_powertools.utilities.data_classes.shared_functions import (
     base64_decode,
     get_header_value,
     get_query_string_value,
 )
 
 
-class VPCLatticeEvent(BaseProxyEvent):
+class VPCLatticeEventBase(BaseProxyEvent):
```

Comment:
Use Base as a prefix to ease search and standards.

```suggestion
class BaseVPCLatticeEvent(BaseProxyEvent):
```